tests/artifacts/javascript/e2eTestsHelpers.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
you don't need this, since it's loaded into an ES module, which is always 'strict'